scripts/pg-drop

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/env python3
+
+#####################################################################
+#
+# pg_drop is a tool for displaying ingress pg dropped packet stats.
+#
+#####################################################################
+import _pickle as pickle
+import argparse
+import json
+import os
+import sys
+from collections import OrderedDict
+
+import swsssdk
+from natsort import natsorted
+from tabulate import tabulate
+
+# mock the redis for unit test purposes #
+try:
+    if os.environ["UTILITIES_UNIT_TESTING"] == "2":
+        modules_path = os.path.join(os.path.dirname(__file__), "..")
+        tests_path = os.path.join(modules_path, "tests")
+        sys.path.insert(0, modules_path)
+        sys.path.insert(0, tests_path)
+        import mock_tables.dbconnector
+
+except KeyError:
+    pass
+
+STATUS_NA = 'N/A'
+
+COUNTER_TABLE_PREFIX = "COUNTERS:"
+
+COUNTERS_PORT_NAME_MAP = "COUNTERS_PORT_NAME_MAP"
+COUNTERS_PG_NAME_MAP = "COUNTERS_PG_NAME_MAP"
+COUNTERS_PG_PORT_MAP = "COUNTERS_PG_PORT_MAP"
+COUNTERS_PG_INDEX_MAP = "COUNTERS_PG_INDEX_MAP"
+
+def get_dropstat_dir():
+    dropstat_dir_prefix = '/tmp/dropstat'
+    return "{}-{}/".format(dropstat_dir_prefix, os.getuid())
+
+class PGdropstat(object):
```

Comment:
is there a convention where each letter of a word in the name should be in capital letter such as PgDropStat?
please check if all other counters use Stat and not Counter/Count